src/helpers/getTvUtils.ts

```diff
@@ -133,6 +133,8 @@ const getDiscoverTv = async ({
 
     const data = (await response.json()) as TvResults;
     if (!data.results || data.results.length < 1) return null;
+    const dataWithType: TvData[] = data.results.map((d) => ({ ...d, media_type: 'tv' }));
```

Comment:
Is the `convertResultsToShowType` function not handling the `media_type` appropriately? why?

Reply:
it assumes that `data.results` has a `media_type` field. This TMDB request doesn't return that so I am manually adding it in.